### DISCUSSION TITLE
How to build JSBSIM with Cygwin on windows ?

### DISCUSSION THREAD

**Discussion by moreba1** [JSBSim-Team/jsbsim#350] [Q&A]
Hi

How we can build jsbsim with cygwin ? 

is related to :
https://github.com/JSBSim-Team/jsbsim/issues/225

**Comment by bcoconni** [comment]
Hi @moreba1,

JSBSim is using CMake and `make` to build. In theory, it should work with Cygwin as well. The process to build [JSBSim with CMake is described in our developer docs](https://github.com/JSBSim-Team/jsbsim/blob/master/doc/DevelopersDocs.md#building-with-cmake).

Let us know if the building process fails for any reason, we will gladly answer your questions (do not forget to provide the error messages you get when reporting an issue).

**Comment by moreba1** [comment]
https://github.com/PX4/PX4-Autopilot/issues/16294

**Comment by bcoconni** [comment]
This issue has been moved to "Discussions". Now that this new feature is available in GitHub, it has been decided to move issues related to support requests and questions to "Discussions" and restrict "Issues" to bugs and feature requests.
This change is only motivated by project organization and will not affect the support provided. We'll gladly continue to answer your questions 😄.
